src/main/java/uk/gov/ida/eventemitter/EventSender.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.ida.eventemitter;
+
+import com.amazonaws.Response;
+
+public interface EventSender {
+
+    Response<Void> sendAuthenticated(final Event event, final String encryptedEvent) throws java.io.UnsupportedEncodingException;
```

Comment:
Can we replace `java.io.UnsupportedEncodingException` with `UnsupportedEncodingException`?